src/steps/os/linux.rs

```diff
@@ -574,6 +574,15 @@ pub fn run_pihole_update(sudo: Option<&PathBuf>, run_type: RunType) -> Result<()
     run_type.execute(sudo).arg(pihole).arg("-up").check_run()
 }
 
+pub fn run_protonup_update(ctx: &ExecutionContext) -> Result<()> {
+    let protonup = require("protonup")?;
+
+    print_separator("protonup");
+
+    ctx.run_type().execute(protonup).check_run()?;
```

Comment:
This seems to be missing support for `--yes` flag (non-interactive mode).

Please see issue here #559

Reply:
You are right, it is not implemented yet